### DISCUSSION TITLE
Importing latest version of PyAEDT

### DISCUSSION THREAD

**Discussion by Abhi-SimuTech** [ansys/pyaedt#3744] [Q&A] [answered]
Hi,

Does adding the following guarantees that I'm using the latest stable release of PyAEDT for my script? Otherwise, what shall I do to make sure!

sys.path.append(r'c:\ansysdev\git\repos\pyaedt')
import pyaedt

Regards
Abhi

**Comment by Samuelopez-ansys** [comment, marked as answer]
Indeed.

The lastest version that you have under this path. If you need the latest version you will need to update this branch, this has to be done manually with git commands.

Thanks

**Comment by MaxJPRey** [comment]
@Abhi-SimuTech ,
In addition of @Samuelopez-ansys solution, I would recommend to not add PyAEDT to your path if you do not intend to make any modification to PyAEDT source code. This will avoid some confusion between packages later on.
If so, just use this command `pip install pyaedt --upgrade `in a command line prompt.
